libusb.go

```diff
@@ -171,11 +180,19 @@ type libusbIntf interface {
 // libusbImpl is an implementation of libusbIntf using real CGo-wrapped libusb.
 type libusbImpl struct{}
 
-func (libusbImpl) init() (*libusbContext, error) {
+func (libusbImpl) init(flags ...libusbOpt) (*libusbContext, error) {
 	var ctx *C.libusb_context
 	if err := fromErrNo(C.libusb_init(&ctx)); err != nil {
 		return nil, err
 	}
+
+	for _, flag := range flags {
+		switch flag {
+		case LIBUSB_OPTION_NO_DEVICE_DISCOVERY:
+			C.gousb_disable_device_discovery(ctx)
```

Comment:
this call can return an error, you need to add fromErrNo(...) wrapper around it and check for nil